src/test/java/org/smartregister/converters/ClientConverterTest.java

```diff
@@ -33,7 +33,11 @@ public void testConvertToPatientResource() {
 		assertEquals(patient.getName().get(0).getFamily().getValue(),client.getLastName());
 		assertEquals(patient.getName().get(0).getText().getValue(),client.fullName());
 		assertEquals(patient.getId(),client.getBaseEntityId());
-		//TODO : Add assertion on remaining properties
+		int identifiersSize = client.getRelationships().entrySet().size() + client.getAttributes().entrySet().size() + client.getIdentifiers().entrySet().size();
+		assertEquals(patient.getIdentifier().size(),identifiersSize);
+		assertEquals(patient.getBirthDate().getValue().toString(),client.getBirthdate().toString("YYYY-MM-DD"));
+		assertEquals(patient.getDeceased().as(com.ibm.fhir.model.type.DateTime.class).getValue().toString(),client.getDeathdate().toString("yyyy-MM-dd'T'HH:mm'Z'"));
+		//TODO : Individual assertions on relationships, attributes and identifiers
```

Comment:
Test number of size of   relationships, attributes and identifiers converted is accurate and also assert one item from each category

Reply:
Done